frontend/src/actionCreators/usersActionCreator.tsx

```diff
@@ -98,4 +98,45 @@ export const getUsersAction = (params: API.IPaginateParams) => (dispatch: Functi
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchUserDetail = (id: API.IUserDetailParams) => (dispatch: Function) => {
+  console.log('BELOW HERE');
+  dispatch(request(reducerTypes.GET_USER_DETAIL));
+  dispatch(showLoading());
+  return CustomAxios()
+    .get(ENVIRONMENT.apiUrl + API.USER_DETAIL(id), createRequestHeader())
+    .then((response: AxiosResponse) => {
+      dispatch(success(reducerTypes.GET_USER_DETAIL));
+      dispatch(adminActions.storeUserDetail(response.data));
+      console.log(response.data);
+      dispatch(hideLoading());
+    })
+    .catch(() => dispatch(error(reducerTypes.GET_USER_DETAIL)))
+    .finally(() => dispatch(hideLoading()));
+};
+
+export const toUpdateUserRequest = (values: any) => {
```

Comment:
I wonder if the backend needs to be updated to handle some of the data not being sent (i.e. username, middlename, isDisabled, emailVerified, ...)?

Reply:
I was wondering if I should add middle name to the edit options.. isDisabled currently sends on change though. And username is also sent to comply with the PUT request but it remains unchanged of course. But I do need to include emailVerified, forgot to add it in, on it now. Forgot to remove this toUpdateUserRequest as it is outdated and no longer used so it is misleading!

Reply:
Or I think I misread that, do you mean as in to handle a PATCH request if there are only certain fields changing?

Reply:
Some of our backend api's are expecting the entire model to be sent from the frontend, not just the fields that are changing. For example, the update parcel workflow gets the model, and then resends that entire model to the backend when the update is saved, regardless of any changes being made.